kzg_prover/src/circuits/utils.rs

```diff
@@ -313,26 +307,27 @@ pub fn verify_opening<
     E: EncodedChallenge<Scheme::Curve>,
     T: TranscriptReadBuffer<&'a [u8], Scheme::Curve, E>,
     Strategy: VerificationStrategy<'params, Scheme, V, Output = Strategy>,
+    const N_POINTS: usize,
```

Comment:
Does `verify_opening` need to be public?

Reply:
No, will remove the modifier.